src/drivers/optical_flow/CMakeLists.txt

```diff
@@ -34,3 +34,4 @@
 add_subdirectory(paw3902)
 add_subdirectory(pmw3901)
 add_subdirectory(px4flow)
+add_subdirectory(upflow)
```

Comment:
Should we name this more descriptively like `upflow_lc302`?

Reply:
I think If they have new module, they would continue use this protocol.